server/sockets/socket-server.js

```diff
@@ -55,10 +55,42 @@ io.on('connection', (socket) => {
     if (people.length > 1) {
       people.allowed = 1;  // setting var for game started true
       io.in(data.roomname).emit('usersList', usernames)       // emitting usernames list to clients
+      let players = people.length                     //no of players
+      //shuffled cards array
+      for (var a=[],i=0;i<54;++i) a[i]=i;
```

Comment:
I am not sure if this can account for multiple decks (when number of players is > 4)